config/redis-openvas.conf

```diff
@@ -106,7 +106,7 @@ tcp-backlog 511
 # incoming connections. There is no default, so Redis will not listen
 # on a unix socket when not specified.
 #
-unixsocket /run/redis-openvas/redis.sock
+unixsocket /run/redis/redis.sock
```

Comment:
When we change the socket location the install documentation does not match anymore: https://greenbone.github.io/docs/latest/21.04/source-build/index.html#setting-up-the-redis-data-store

Reply:
That's 21.04 there it is still accurate?

Reply:
Anyway changed it back and adapted workflow